### DISCUSSION TITLE
Organization members can still login if organization is disabled.

### DISCUSSION THREAD

**Discussion by ashestak-sc** [keycloak/keycloak#33773] [Q&A] [answered]
Hi. There is a question related to the new 'ORGANIZATION' feature.
If user was created in keycloak (not by identity provider), then we add him to some organization, disable this organization, and try to login => he can do it. Organization disabling does nothing with members in such a case?

**Comment by codespearhead** [comment, marked as answer]
Does this [1] answer your question?

[1] https://github.com/keycloak/keycloak/blob/04e8508aaa48592b202a7d5a1c8737065b8435fa/docs/documentation/server_admin/topics/organizations/managing-organization.adoc?plain=1#L78-L80
